### ISSUE TITLE
No such namespace: thi.ng.common.data.core

### ISSUE THREAD

**Issue by jackrusher** [thi-ng/morphogen#2]
It looks like `[morphogen "0.1.1"]` depends upon but does not include `thi.ng/common`, resulting in the above error when one attempts to `require` it. Adding `[thi.ng/common "0.3.1"]` to one's deps resolves the problem.

**Comment by postspectacular** [comment]
Hi @jackrusher - oops.. that's a regression from thi.ng/geom, which is not including the common lib anymore (in fact common has been split up into different libs to avoid turning into a kitchen sink). will build a new 0.2.0-SNAPSHOT of morphogen and let you know when ready...

**Comment by jackrusher** [comment]
Grand. I'm guessing the new SNAPSHOT will also fix what look to be protocol incompatibilities with `mg/save-mesh` as well. :)

**Comment by postspectacular** [comment]
yes, it should - but alas it won't happen today (will try tomorrow eve) - got another workshop to prepare (this weekend) :)
